jdi-light-vuetify-tests/src/main/java/io/github/com/pages/CheckboxesPage.java

```diff
@@ -35,4 +35,8 @@ public class CheckboxesPage extends VuetifyPage {
     @UI("#InlineTextFieldCheckboxes .v-input--checkbox")
     @WaitAfterAction(1)
     public static List<Checkbox> inlineTextFieldCheckboxes;
+
+    @UI("#InlineTextFieldCheckboxes .v-label")
+    @WaitAfterAction(1)
+    public static List<Checkbox> inlineTextField;
```

Comment:
inlineTextField is not Checkbox (label -> input)

I suppose it's better to find this element using inlineTextFieldCheckboxes (it's a part of InlineTextFieldCheckboxes)